### PR TITLE
Add a security and privacy section

### PR DESCRIPTION
**Pull request by takumif**

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
    source.
 1. How to handle messaging.
 
+## Security and privacy considerations
+
+See [the corresponding section](
+https://w3c.github.io/presentation-api/#security-and-privacy-considerations) of
+the Presentation API spec. Below, we list characteristics unique to site
+initiated mirroring.
```
Comment (markafoltz) [inline explainer.md]: There are also some aspects that do not apply to site-initiate mirroring, like the discussion of DIAL URLs and connection from multiple controllers.  It would be worth mentioning what aspects of the generic API usage don't apply here.

File: explainer.md [hunk 2]
```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
...
+
+### Personally identifiable information
+
+By [monitoring the list of available presentation displays](
+https://w3c.github.io/presentation-api/#monitoring-the-list-of-available-presentation-displays),
+a website can determine whether the user has devices compatible with mirroring,
```
Comment (markafoltz) [inline explainer.md]: Let's use "presentation displays" instead of "devices" for consistency with the spec.

File: explainer.md [hunk 3]
```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
...
+which is one new bit of information that can be used for fingerprinting. This
+bit however would likely strongly correlate with whether the user has devices
```
Comment (markafoltz) [inline explainer.md]: I found this sentence hard to follow, especially if the reader doesn't know what the "existing modes" are.

Maybe:

"Nearly all presentation displays support mirroring, so this additional information does not add significantly to the fingerprinting surface of the API."

File: explainer.md [hunk 4]
```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
...
+compatible with the existing modes of Presentation API, which is a bit that is
+already exposed.
+
+### Cross-origin access
+
+The proposed API will not grant access to the captured video stream to either
```
Comment (markafoltz) [inline explainer.md]: This API doesn't support capture, so the reference to "captured" is misleading.  Maybe "...access to the mirrored content..." ?

File: explainer.md [hunk 5]
```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
...
+the controlling context initiating site initiated mirroring or other controlling
+contexts that join (i.e. `reconnect()` to) an existing mirroring session, so no
```
Comment (markafoltz) [inline explainer.md:218]: I assumed mirroring sessions would not support `reconnect()`.  If there is a use case, maybe we can add that as a later feature/PR.

**Review by markafoltz — CHANGES_REQUESTED** [review]

File: explainer.md (hunk 5 above)
Comment (takumif) [reply explainer.md:218]: I was thinking that the default expectation would be that you can reconnect() to any presentation, and wasn't seeing a strong reason to explicitly disallow that for mirroring.

Not that I expect reconnect() to be used much though. You could reconnect() then terminate() but I think that's about it.

File: explainer.md (hunk 1 above)
Comment (takumif) [reply explainer.md]: Added mentions of DIAL, private browsing, and messaging.
I don't know if there's a need to specify that reconnect() from multiple controllers should be disallowed. Reconnecting to a mirroring session would just be uninteresting as mentioned below. I didn't find any mentions of reconnect() not being possible for 1UA in the spec.

**Comment by takumif** [comment]
Thanks for the comments, please take another look.

File: explainer.md (hunk 5 above)
Comment (markafoltz) [reply explainer.md:218]: That sounds fine.

File: explainer.md [hunk 6]
```diff
@@ -1,4 +1,5 @@
-# Site Initiated Mirroring API
+Below, we list characteristics unique to site
+initiated mirroring.# Site Initiated Mirroring API
```
Comment (markafoltz) [inline explainer.md]: Should this header go on its own line?

**Review by markafoltz — APPROVED** [review]

**Comment by markafoltz** [comment]
Merging as all comments have been resolved.  Thanks for the update @takumif
